app/controllers/users_controller.rb

```diff
@@ -1,2 +1,39 @@
 class UsersController < ApplicationController
+  before_action :set_user, only: %i(show)
+  def index
+    @pagy, @users = pagy(User.new_user,
+                         item: Settings.users_on_page)
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def edit; end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      # @user.avatar.attach(params[:avatar])
+      session[:user_id] = @user.id
+      redirect_to root_path
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show; end
+  private
+  def set_user
```

Comment:
```suggestion
  def show; end

  private

  def set_user
```
Coding convention đấy